GhostNetwork.Content/Media.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace GhostNetwork.Content;
+
+public class Media
+{
+    public Media(Guid id, string link)
+    {
+        Id = id;
+        Link = link;
+    }
+
+    public Guid Id { get; set; }
```

Comment:
It seems to me I’ve already asked to remove Id from Media

What’s use case for this field?